pages/assessment.tsx

```diff
@@ -54,36 +52,45 @@ const Assessment: NextPage = () => {
 
   const startIndexBatch = currentPage * maxItemsPerPage - maxItemsPerPage;
   const endIndexBatch = currentPage * maxItemsPerPage;
-  
-   // to make sure setAssessments is being set, otherwise it is empty
-   useEffect(() => {
+
+  // to make sure setAssessments is being set, otherwise it is empty
+  useEffect(() => {
     if (assessments.length == 0) {
       setAssessments(p);
     }
   }, [assessments.length, p]);
 
   const changePage = (direction: string): void => {
-      if (direction == "back") {
+    if (direction == "back") {
       setCurrentPage(currentPage - 1);
     } else if (direction == "next") {
       appendReAssessments(assessments.slice(startIndexBatch, endIndexBatch));
       setCurrentPage(currentPage + 1);
     }
   };
 
-  const appendReAssessments = (batch: AssessmentType[]) =>{
+  const appendReAssessments = (batch: AssessmentType[]) => {
     const assessment = chooseCorrelatedAssessment(batch);
     // if an outlier was returned and the reAssessment-list is not full (over 20%), then append (if it is not there already)
-    if (assessment!= null && reAssessments.length < Math.floor(assessments.length*0.2)) {
-        if (reAssessments.filter(a => a.assessmentId == assessment.assessmentId).length < 1 ) {
+    if (
+      assessment != null &&
+      reAssessments.length < Math.floor(assessments.length * 0.2)
+    ) {
+      if (
+        reAssessments.filter((a) => a.assessmentId == assessment.assessmentId)
+          .length < 1
+      ) {
```

Comment:
This is probably just prettier formatting, right?

Reply:
Yup

Reply:
It is because when I first pushed the code, I was working from home on my macbook where my prettier extension is not working as expected...

Reply:
yeah! Nice with a verification:)